stage6/09-sense-emu/00-run.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash -e
+
+on_chroot << EOF
+pip3 install sense-emu
+EOF
+
+icon="sense_emu_gui.svg"
+desktop_path=~/.local/share/applications
```

Comment:
"~" means that, whatever the user is - 'analog' or 'root' in Kuiper case, will need to have "./.local/share/applications" folder structure in its home.

Reply:
Updated with [fix](https://github.com/analogdevicesinc/adi-kuiper-gen/pull/122/commits/a4c287cce46c887aee7cc0db2bdb76ee6157f217)